### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by etrodrigues**
New entry
| Floridi | [Machine Unlearning: its nature, scope, and importance for a “delete culture”](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4455976) | Philosophy & Technology |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 ### 2023
 | Author(s) | Title | Venue |
 |:--------- | ----- | ----- |
+| Floridi | [Machine Unlearning: its nature, scope, and importance for a “delete culture”](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4455976) | SSRN |
```
Comment (jjbrophy47) [inline README.md:22]: Can you please change "SSRN" to the actual journal name, I think in this case it is "Philosophy & Technology".

**Review by jjbrophy47 — CHANGES_REQUESTED** [review]
Hi etrodrigues! I left one small comment requesting a change, but after that I'll approve the PR, thanks so much for contributing!

File: README.md (hunk 1 above)
Comment (etrodrigues) [reply README.md:22]: Done! Thanks!

**Comment by jjbrophy47** [comment]
Looks great, thanks!

**Pull request merged**